atomicapp/nulecule/lib.py

```diff
@@ -64,46 +63,21 @@ def load_config(self, config, ask=False, skip_asking=False):
             None
         """
         for param in self.params:
-            value = config.get(self.namespace, {}).get(param[NAME_KEY]) or \
-                config.get(GLOBAL_CONF, {}).get(param[NAME_KEY])
+            value = config.get(param[NAME_KEY], scope=self.namespace)
```

Comment:
@rtnpro - would it be reasonable in the future (not for the PR) for us to maybe rename `namespace` from this class to `scope`?